niworkflows/interfaces/bids.py

```diff
@@ -911,6 +912,12 @@ def _run_interface(self, runtime):
                 else:
                     raise FileNotFoundError("Expected to find '%s' to copy" % source)
 
+            if dest.exists() and self.inputs.minimum_fs_version == "7.0.0":
+                label = dest / 'label' / 'rh.FG1.mpm.vpnl.label'  # new in FS7
```

Comment:
You're running this on all spaces, but it only exists in `fsaverage`:

```
$ ls /usr/local/freesurfer/7.2.0/subjects/fsaverage*/label/rh.FG1*
/usr/local/freesurfer/7.2.0/subjects/fsaverage/label/rh.FG1.mpm.vpnl.label
```

I would probably only bother replacing that one. The difference in `fsaverage5` is irrelevant to us:

```diff      
diff --color -r /usr/local/freesurfer/6.0.1/subjects/fsaverage5/stats/subject.aseg.mean.stats /usr/local/freesurfer/7.2.0/subjects/fsaverage5/stats/subject.aseg.mean.stats
1c1
< Measure:mean	Left-Cerebral-White-Matter	Left-Cerebral-Cortex	Left-Lateral-Ventricle	Left-Inf-Lat-Vent	Left-Cerebellum-White-Matter	Left-Cerebellum-Cortex	Left-Thalamus-Proper	Left-Caudate	Left-Putamen	Left-Pallidum	3rd-Ventricle	4th-Ventricle	Brain-Stem	Left-Hippocampus	Left-Amygdala	CSF	Left-Accumbens-area	Left-VentralDC	Left-vessel	Left-choroid-plexus	Right-Cerebral-White-Matter	Right-Cerebral-Cortex	Right-Lateral-Ventricle	Right-Inf-Lat-Vent	Right-Cerebellum-White-Matter	Right-Cerebellum-Cortex	Right-Thalamus-Proper	Right-Caudate	Right-Putamen	Right-Pallidum	Right-Hippocampus	Right-Amygdala	Right-Accumbens-area	Right-VentralDC	Right-vessel	Right-choroid-plexus	5th-Ventricle	WM-hypointensities	Left-WM-hypointensities	Right-WM-hypointensities	non-WM-hypointensities	Left-non-WM-hypointensities	Right-non-WM-hypointensities	Optic-Chiasm	CC_Posterior	CC_Mid_Posterior	CC_Central	CC_Mid_Anterior	CC_Anterior
---
> Measure:mean	Left-Cerebral-White-Matter	Left-Cerebral-Cortex	Left-Lateral-Ventricle	Left-Inf-Lat-Vent	Left-Cerebellum-White-Matter	Left-Cerebellum-Cortex	Left-Thalamus	Left-CaudateLeft-Putamen	Left-Pallidum	3rd-Ventricle	4th-Ventricle	Brain-Stem	Left-Hippocampus	Left-Amygdala	CSF	Left-Accumbens-area	Left-VentralDC	Left-vessel	Left-choroid-plexusRight-Cerebral-White-Matter	Right-Cerebral-Cortex	Right-Lateral-Ventricle	Right-Inf-Lat-Vent	Right-Cerebellum-White-Matter	Right-Cerebellum-Cortex	Right-Thalamus	Right-Caudate	Right-Putamen	Right-Pallidum	Right-Hippocampus	Right-Amygdala	Right-Accumbens-area	Right-VentralDC	Right-vessel	Right-choroid-plexus	5th-Ventricle	WM-hypointensities	Left-WM-hypointensities	Right-WM-hypointensities	non-WM-hypointensities	Left-non-WM-hypointensitiesRight-non-WM-hypointensities	Optic-Chiasm	CC_Posterior	CC_Mid_Posterior	CC_Central	CC_Mid_Anterior	CC_Anterior
diff --color -r /usr/local/freesurfer/6.0.1/subjects/fsaverage5/stats/subject.aseg.volume.stats /usr/local/freesurfer/7.2.0/subjects/fsaverage5/stats/subject.aseg.volume.stats
1c1
< Measure:volume	Left-Cerebral-White-Matter	Left-Cerebral-Cortex	Left-Lateral-Ventricle	Left-Inf-Lat-Vent	Left-Cerebellum-White-Matter	Left-Cerebellum-Cortex	Left-Thalamus-Proper	Left-Caudate	Left-Putamen	Left-Pallidum	3rd-Ventricle	4th-Ventricle	Brain-Stem	Left-Hippocampus	Left-Amygdala	CSF	Left-Accumbens-area	Left-VentralDC	Left-vessel	Left-choroid-plexus	Right-Cerebral-White-Matter	Right-Cerebral-Cortex	Right-Lateral-Ventricle	Right-Inf-Lat-Vent	Right-Cerebellum-White-Matter	Right-Cerebellum-Cortex	Right-Thalamus-Proper	Right-Caudate	Right-Putamen	Right-Pallidum	Right-Hippocampus	Right-Amygdala	Right-Accumbens-area	Right-VentralDC	Right-vessel	Right-choroid-plexus	5th-Ventricle	WM-hypointensities	Left-WM-hypointensities	Right-WM-hypointensities	non-WM-hypointensities	Left-non-WM-hypointensities	Right-non-WM-hypointensities	Optic-ChiasmCC_Posterior	CC_Mid_Posterior	CC_Central	CC_Mid_Anterior	CC_Anterior	BrainSegVolIntraCranialVol
---
> Measure:volume	Left-Cerebral-White-Matter	Left-Cerebral-Cortex	Left-Lateral-Ventricle	Left-Inf-Lat-Vent	Left-Cerebellum-White-Matter	Left-Cerebellum-Cortex	Left-Thalamus	Left-Caudate	Left-Putamen	Left-Pallidum	3rd-Ventricle	4th-Ventricle	Brain-Stem	Left-Hippocampus	Left-Amygdala	CSF	Left-Accumbens-area	Left-VentralDC	Left-vessel	Left-choroid-plexus	Right-Cerebral-White-Matter	Right-Cerebral-Cortex	Right-Lateral-Ventricle	Right-Inf-Lat-Vent	Right-Cerebellum-White-Matter	Right-Cerebellum-Cortex	Right-Thalamus	Right-Caudate	Right-Putamen	Right-Pallidum	Right-Hippocampus	Right-Amygdala	Right-Accumbens-areaRight-VentralDC	Right-vessel	Right-choroid-plexus	5th-Ventricle	WM-hypointensities	Left-WM-hypointensities	Right-WM-hypointensities	non-WM-hypointensities	Left-non-WM-hypointensities	Right-non-WM-hypointensities	Optic-Chiasm	CC_Posterior	CC_Mid_Posterior	CC_Central	CC_Mid_Anterior	CC_Anterior	BrainSegVol	IntraCranialVol
```